<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DA16010730" ignored="2" name="Default test" tests="3" failures="1" timestamp="09 Nov 2018 19:28:35 GMT" time="32.473" errors="0">
  <testcase name="loginCheck" time="9.002" classname="caseOfTest.LoginTest"/>
  <testcase name="openCreateQuote" time="23.405" classname="caseOfTest.LoginTest"/>
  <testcase name="createQuoteWithData" time="0.009" classname="caseOfTest.LoginTest">
    <failure type="java.lang.IllegalArgumentException" message="Cannot find elements when the XPath is null.">
      <![CDATA[java.lang.IllegalArgumentException: Cannot find elements when the XPath is null.
at org.openqa.selenium.By.xpath(By.java:113)
at caseOfTest.LoginTest.createQuoteWithData(LoginTest.java:73)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createQuoteWithData -->
  <testcase name="@AfterSuite tearDown" time="0.0" classname="testBase.GroundFloor">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase name="@BeforeSuite setUp" time="0.0" classname="testBase.GroundFloor">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
</testsuite> <!-- Default test -->
